src/utils/curl.js

```diff
@@ -0,0 +1,25 @@
+import pkg from '../../package.json';
+
+const defaultHeaders = {
+  'User-Agent': `{pkg.name}/${pkg.version}`,
+};
+
+function curl(url, { headers = {} } = {}) {
+  const mergedHeaders = Object.etries({
```

Comment:
This needs to be fixed
```suggestion
  const mergedHeaders = Object.entries({
```